pubnub/features.py

```diff
@@ -0,0 +1,20 @@
+from os import getenv
+from pubnub.exceptions import PubNubException
+
+flags = {
+    'PN_ENABLE_ENTITIES': getenv('PN_ENABLE_ENTITIES', False)
+}
+
+
+def feature_flag(flag):
+    def not_implemented(*args, **kwargs):
```

Comment:
I am not sure if these functions must be declared inside decorator function, please check it :) 
If not I think it will be much more readable if they will be outside decorator function

Reply:
I've declared it inside of the decorator function because i just need a callable to return and don't need it outside of this scope. It seemed to me more logical this way